docs/docs/03-language-reference.md

```diff
@@ -1543,11 +1543,18 @@ f(1, 2, field1: 3, field2: 4);
 // f(1, 2, field1: 3); // can't do this, partial expansion is not allowed
 ```
 
-#### 3.6.3 Roadmap
-
-The following features are not yet implemented, but we are planning to add them in the future:
-
-* Variadic arguments (`...args`) - see https://github.com/winglang/wing/issues/125 to track.
+#### 3.6.3 Variadic Arguments
+If the last argument of a function type is the `...args` keyword followed by an
+`Array` type, then the function accepts typed variadic arguments. The container of variadic
+arguments is accessible with the `args` key like a normal array instance.
```

Comment:
I believe this works with any parameter name (not just `args`)

Reply:
Right, thanks. Addressed.